source/includes/_applications.md

```diff
@@ -369,3 +369,33 @@ In order to delete a closed or hired Job Application you must use the following
 `DELETE /v1/companies/{companyId}/applications/{jobApplicationId}`
 
 If the request succeeds, the api will return a 200 response with no body.
+
+## Edit a JobApplication
+
+This endpoint is used to edit a job application's information. It can also be used to edit and delete associated files and profile pictures from storage.
+To do this provide an empty array in the `files` field. When updating the job application any existing file that is left out in the update request, will be deleted.
+Also if you send a request without applicantImage, the associated Image will be deleted.
+
+### HTTP Request
+
+`PUT /v1/companies/{companyId}/applications/{jobApplicationId}`
+
+> this is an example request body
+
+```json
+{
+  "firstName": "Test",
+  "lastName": "Applicant",
+  "email": "test.applicant@firstbird.com",
+  "phoneNumber": "123456789",
+  "url": "https://somesocialmediaplattform.com/users/TestApplicant",
+  "files": [
+    {
+      "id": "c91419ea-320d-40df-8d60-37499f20c589"
```

Comment:
please use 00000000-0000-0000-0000-00000000000X (x=0, then x =1, etc.)

Reply:
^ This, otherwise LGTM. I think you missed it since you updated all the other ones.